Corona-Warn-App/src/main/java/de/rki/coronawarnapp/submission/task/SubmissionTask.kt

```diff
@@ -48,9 +51,13 @@ class SubmissionTask @Inject constructor(
             Timber.tag(TAG).d("Running with arguments=%s", arguments)
             arguments as Arguments
 
-            if (arguments.checkUserActivity && hasRecentUserActivity()) {
-                Timber.tag(TAG).w("User has recently been active in submission, skipping submission.")
-                return Result(state = Result.State.SKIPPED)
+            if (arguments.checkUserActivity) {
+                if (hasRecentUserActivity()) {
+                    Timber.tag(TAG).w("User has recently been active in submission, skipping submission.")
+                    return Result(state = Result.State.SKIPPED)
+                } else {
+                    analyticsKeySubmissionCollector.reportSubmittedInBackground()
```

Comment:
Submission may still abort for other reasons, should we call `reportSubmittedInBackground` later when we are sure that it was submitted?